tool/dart_dev/config.dart

```diff
@@ -0,0 +1,7 @@
+import 'package:dart_dev/dart_dev.dart';
+
+final Map<String, DevTool> config = {
+  'analyze': AnalyzeTool()..analyzerArgs,
+  'format': FormatTool(),
+  'test': TestTool(),
+};
```

Comment:
I think if you add a dependency on `dart_dev_workiva`, we could could simplify this, which might be nice to do.

```suggestion
import 'package:dart_dev/dart_dev.dart';
import 'package:dart_dev_workiva/dart_dev_workiva.dart';

final config = workivaConfig;
```

Reply:
Wait, I'm pretty sure we can't do this - https://pub.dev/packages/r_tree

Reply:
Oh, nice catch. I forgot this is an OSS repo. 🤕